charts/cronjob/README.md

```diff
@@ -63,6 +63,10 @@ The chart can be customized using the following configurable parameters:
 | `concurrency` | specifies how to treat concurrent executions of a job created by this cron job | `Forbid` |
 | `successfulJobsHistoryLimit` | Cron job successfulJobsHistoryLimit | 5 |
 | `failedJobsHistoryLimit` | Cron job failedJobsHistoryLimit | 5 |
+| `externalConfig.enabled` | Enable supplying external configuration file to the cronjob | false |
+| `externalConfig.path` | Path to the external config file, which can be supplied to the cronjob | /path/to/file |
```

Comment:
Isn't is rather `Path to the external directory which contains configuration file (which can be optionally supplied to the cronjob)`?

Reply:
Is `/path/to/file` default value pr rather an example? Maybe we should introduce additional 2 columns?

`| Parameter | Description | Required | Default | Example |`

Reply:
Maybe not in this PR but it would be nice if one could supply multiple files. For example all 3 parameters `path`, `configFileName` and `content` could contain multiple comma separated values.

Reply:
https://github.com/commercetools/k8s-charts/issues/24